subworkflows/local/generate_downstream_samplesheets/main.nf

```diff
@@ -0,0 +1,49 @@
+//
+// Subworkflow with functionality specific to the nf-core/createtaxdb pipeline
+//
+
+workflow GENERATE_DOWNSTREAM_SAMPLESHEETS {
+    take:
+    ch_databases
+
+    main:
+    ch_header  = Channel.empty()
+    format     = 'csv' // most common format in nf-core
+    format_sep = ','
+
+    // TODO --
+    // Make your samplesheet channel construct here depending on your downstream
+    // pipelines
+    if ( params.generate_pipeline_samplesheets == 'taxprofiler' ) {
+        format = 'csv'
+        format_sep = ','
```

Comment:
Does this need to be defined twice?

Reply:
No, I was leaving it in as an example but can remove this now :)